docs/earthly-command/earthly-command.md

```diff
@@ -244,6 +244,10 @@ Also available as an env var setting: `EARTHLY_INTERACTIVE=true`.
 
 Enable interactive debugging mode. By default when a `RUN` command fails, earthly will display the error and exit. If the interactive mode is enabled and an error occurs, an interactive shell is presented which can be used for investigating the error interactively. Due to technical limitations, only a single interactive shell can be used on the system at any given time.
 
+##### `--strict`
+
+Disallow usage of features that may create unreproducible builds.
```

Comment:
```suggestion
Disallow usage of features that may create unrepeatable builds.
```

Reply:
This is our help text for the command. I'll update that too.